src/calculateRentalCost.js

```diff
@@ -4,7 +4,9 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  console.log('No solution at all');
+
+  return 1 + 1;
 }
```

Comment:
The current implementation of the function `calculateRentalCost` does not follow the task requirements. It should calculate the total cost based on the number of days the car is rented using the specified pricing strategy. Make sure to implement the logic as described in the task, using constants for the daily rate, long term rental days threshold, long term discount, short term rental days threshold, and short term discount.